modules/power-monitoring-accessing-dashboards-developer.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-accessing-dashboards-developer_{context}"]
+= Accessing {PM-shortname} dashboards as a developer
+
+You can access {PM-shortname} dashboards from the *Developer* perspective of the {product-title} web console.
```

Comment:
If you log in as a cluster-admin, you need to click "Administrator" and select "Developer" to switch to the **Developer** perspective.  I don't know if it's necessary to explain this.

Reply:
This is a procedure module. Before going ahead with the procedure, we need to specify what will be the result for the user using this procedure. Hope that explains why this sentence is used here.